arrays.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>lesson 7 arrays</title>
+</head>
+
+<body>
+<script src="https://travisshears.com/apps/random-wiki/0.0.4--alpha3/src.7b19814e.js"></script>
+<h2 id="firstA" style=color:blue>The first article</h2>
+<h2 id="lastA" style=color:red>The last article</h2>
+<ul id="articlesList"><li></li></ul>
+
+<script>
+    let articles = [];
+    const firstAEl = document.getElementById('firstA');
+    const lastAEl = document.getElementByID('lastA');
+    const ulEl = document.getElementByID('articlesList')
+
+    function handleNewArticle (articles) {
```

Comment:
code inside a function would fail, because this attribute name should be **article**, because in next line you're using it and push it inside articl**s** array.